.github/workflows/workflows copy/index.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: CI Tests
+
+on: 
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  test-affected-lint:
+    uses: ./.github/workflows/lint.yml
```

Comment:
Add newline at end of file

Reply:
I think this will be picked up in tests in future hopefully